deploy/chart/values.yaml

```diff
@@ -0,0 +1,77 @@
+cephfsCtrlpluginSa:
+  serviceAccount:
+    annotations: {}
+cephfsNodepluginSa:
+  serviceAccount:
+    annotations: {}
+controllerManager:
+  kubeRbacProxy:
+    args:
+    - --secure-listen-address=0.0.0.0:8443
+    - --upstream=http://127.0.0.1:8080/
+    - --v=0
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: true
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
+      tag: v0.16.0
+    resources:
+      limits:
+        cpu: 500m
+        memory: 128Mi
+      requests:
+        cpu: 5m
+        memory: 64Mi
+  manager:
+    args:
+    - --health-probe-bind-address=:8081
+    - --metrics-bind-address=127.0.0.1:8080
+    - --leader-elect
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: true
+    env:
+      csiServiceAccountPrefix: ceph-csi-operator-
+      watchNamespace: ""
+    image:
+      repository: quay.io/cephcsi/ceph-csi-operator
+      tag: latest
```

Comment:
We would need a version-specific tag published when a version is released.

Reply:
And how will the csi operator be versioned?

Reply:
@travisn Yes this is only for main branch, we need to adjust the script to change the tag based on the release version

Reply:
@Madhu-1 JFYI #202

Reply:
@subhamkrai i will take care of it, Currently there is no way to automate it, i have to take care of it manually before releases.

Reply:
okay, just wanted to raise here